aws-ssm-parameter/aws-ssm-parameter.json

```diff
@@ -78,15 +78,18 @@
     "primaryIdentifier": [
         "/properties/Name"
     ],
-    "readOnlyProperties": [
-        "/properties/Name",
-        "/properties/Type",
-        "/properties/Value"
+    "writeOnlyProperties": [
```

Comment:
Can you please let me know if the contract tests were working with this readOnlyProperties removal? I remember to have added this readOnlyProperties block to fix the contract test issues.

Reply:
Yes looks like contract test in personal account works with it.